src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-left: 4px;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-image: url('../images/star.svg');
```

Comment:
Ensure that the image 'star.svg' exists in the '../images/' directory and the path is correct. Otherwise, the background image will not display.